shipment_planning_maxoptra/wizard/shipment_maxoptra_schedule_import.py

```diff
@@ -206,6 +196,10 @@ def _group_pickings_by_type(self, maxoptra_deliveries):
                 _("No matching picking found for Order reference \n %s")
                 % ", ".join(list(missing_names))
             )
+        return self._group_to_batch_flow(pickings)
+
+    def _group_to_batch_flow(self, pickings):
```

Comment:
IMO this function could be called `_group_pickings_by_type` and `_group_pickings_by_type` could be called `_group_maxoptra_deliveries_by_type` in order to reflect what each function is doing